### ISSUE TITLE
Make company names underlined

### ISSUE THREAD

**Issue by lalarus** [Klimatbyran/beta#138]
**Description**
In the company list, company names should be underlined. By doing so, users will see that each company row is clickable, making it possible for them to find the detailed company subpage.

**Motivation**
It is currently unclear to the user how to get to the companies' detailed view (subpage). Table rows lack a visual cue indicating that they are clickable.

**Definition of Done**
Company names are underlined. Targeting the right class with `text-decoration: underline` should fix this.

**When should it be done?**
Rather urgent. Important data is currently hidden from the user.

**Comment by Greenheart** [comment]
Good feedback! Thanks, I will take a look at this.

**Comment by Greenheart** [comment]
Should this also happen for municipalities in the list?

**Comment by lalarus** [comment]
Good point! I suggest so, yes, since we do have the same issue there as well.

**Comment by lalarus** [comment]
A future enhancement for large & medium display sizes could be to change row colors on hover (if feasible). Maybe additionally a button stating for example "More details".
Only future thoughts - underlining the names is a good first step which definitely solves the problem!
